ui-components/src/blocks/Tooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef } from "react"
+import useKeyPress from "../helpers/useKeyPress"
+import "./Tooltip.scss"
+
+export interface TooltipProps {
+  className?: string
+  id: string
+  text: string
+}
+
+export interface TooltipPosition {
+  top: number
+  left: number
+}
+
+const Tooltip: React.FC<TooltipProps> = ({ className, id, text, children }) => {
```

Comment:
We don't use the `React.FC` notation, so let's use the generic functional component notation